craft_parts/utils/deb_utils.py

```diff
@@ -0,0 +1,38 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2015-2022 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""deb-related utilities used by both `packages` and `sources`."""
+
+import subprocess
+from pathlib import Path
+from typing import Callable
+
+from craft_parts import errors
+from craft_parts.utils import os_utils
+
+
+def extract_deb(
```

Comment:
I wouldn´t mind this to be part to the deb source module, I don't see components other than the deb source handler and the deb package handler using deb extraction, and package handling may be spun off as a separate project in the future.

Reply:
That would mean that `packages.deb` would import `sources.deb_source`. Is that ok?

Reply:
I think that having deb_utils is fine